src/main/java/meteordevelopment/meteorclient/utils/entity/effects/StatusEffectBruteForce.java

```diff
@@ -151,8 +151,11 @@ private static void update(int particleColor, LivingEntity entity, EntityEffectC
             int absorptionLevel = Math.round(entity.getAbsorptionAmount() / 4f);
             if (absorptionLevel <= 4) {
                 possibleEntries = new ReferenceOpenHashSet<>(ALL_ENTRIES);
-                possibleEntries.add(ABSORPTION_STRONG);
-                if (absorptionLevel <= 1) possibleEntries.add(ABSORPTION);
+                if (absorptionLevel <= 1) {
+                    possibleEntries.add(ABSORPTION);
+                } else {
```

Comment:
the absorption hearts only give us the lower bound for the absorption level, it should still add both if `absorptionLevel <= 1`

Reply:
Thanks

Reply:
yes im terrible at naming variables :)